src/static/api_documentation.yml

```diff
@@ -32,7 +32,7 @@ paths:
         '503 - Service Unavailable':
           $ref: '#/components/responses/NoCeleryWorkers'
 
-  "/models/generate":
+  "/models/flood/generate":
```

Comment:
I need to change this for the front-end so leave this comment up and I will resolve it in this branch before the PR is completed.